accepted/targeting-packs/targeting-packs.md

```diff
@@ -0,0 +1,159 @@
+# Easy Acquisition of .NET Framework Targeting Packs
+
+**PM** [Immo Landwerth](https://github.com/terrajobst) |
+**Dev** [Daniel Plaisted](https://github.com/dsplaisted)
+
+This proposal is about enabling acquisition of the .NET Framework targeting
+packs via NuGet so that compiling code for a specific version of the .NET
+Framework doesn't require a Windows machine with a machine-wide installation of
+the corresponding targeting pack.
+
+The .NET Framework supports targeting different versions of the platform
+regardless of the .NET Framework version that is installed on the developer's
+machine. This allows developers to target a lower version than what they use
+themselves. This is especially useful for in-place updates where the older
+version of the framework cannot be installed side-by- side.
+
+That mechanism is called *targeting packs*, which is simply a set of assemblies
+that describe the public APIs of a particular version of the .NET Framework.
+Since those assemblies are only used by the compiler, they don't need to contain
+any code, which we call [reference assemblies](https://www.youtube.com/watch?v=EBpY1UMHDY8).
+
+Unfortunately, the targeting packs aren't available by themselves; they are
+redistributed as part of the .NET Framework Developer Packs which bundle both
+the runtime as well as the targeting pack. This design ensures that developers
+can never be in a state where they have a targeting pack that is higher than the
+.NET Framework version they have installed. That design made sense coming from a
+Visual Studio-first perspective but doesn't serve us well today:
+
+* **It's not cross-platform friendly**. The .NET Framework Developer Packs are
+  only available for Windows which makes the targeting packs indirectly Windows-
+  only too.
+
+* **It's not CI friendly**. Requiring a machine-wide install isn't friendly for
+  cloud-hosted CI machines as developers typically lack permissions to install
+  software.
+
+This document proposes to offer the targeting packs via NuGet that they can be
+acquired in a non-impactful way. It builds on the following existing specs:
+
+* [Phillip Carter](https://github.com/cartermp): [Add Target Packs on NuGet](https://github.com/dotnet/core/pull/64)
+* [Daniel Plaisted](https://github.com/dsplaisted): [Reference Assembly NuGet packages](https://gist.github.com/dsplaisted/83d67bbcff9ec1d0aff1bea1bf4ad79a)
+* [Prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages)
+
+## Scenarios and User Experience
+
+* Build a project which is multi-targeted to .NET Core or .NET Standard as well
+  as .NET Framework on Mac OS or Linux using the .NET CLI
+
+* Build a project targeting Mono (which uses the same Target Framework
+  Identifier as .NET Framework) using the .NET CLI
+
+## Requirements
+
+### Goals
```

Comment:
What about PCL profiles? Is that a goal or non-goal? Several popular nuget packages still target PCLs because quite a few customers still use VS2015. Personally, I own several packages that build from .NET SDK projects and multi-target `.netstandard*`, `net4*`, and `portable-*`, and testing on mac+linux is important to us.

Reply:
.NET 2, 3, 3.5 and PCL profiles would be great! Partial support will only result in partial reliance, which is not good enough in many popular packages like Newtonsoft.JSON etc.

Reply:
I don't think that there are reference assemblies / targeting packs for <= 3.5
MSBuild even requires to run a version of MSBuildTaskHost.exe on the system install of the CLR 2 (.NET 2, 3, 3.5) in order to build .NET 3.5 projects.